contracts/external/cw-admin-factory/src/contract.rs

```diff
@@ -39,17 +45,26 @@ pub fn execute(
             instantiate_msg: msg,
             code_id,
             label,
-        } => instantiate_contract(env, info, msg, code_id, label),
+        } => instantiate_contract(deps, env, info, msg, code_id, label),
+        ExecuteMsg::UpdateAdmin { admin } => execute_update_admin(deps, info, admin),
```

Comment:
Nit: current pattern in most of our contracts is that we use `cw-ownable` or Oak / some other auditor will have comments about not having a two-step admin transfer process.

Reply:
yeah hm, that just felt like overkill here.
i'm also open to removing the ability to update at all, and just make this an immutable contract. wdyt?

Reply:
yeah, let's keep it stupid simple. i'm making it immutable

Reply:
> yeah, let's keep it stupid simple. i'm making it immutable

👑